Makefile

```diff
@@ -98,6 +98,16 @@ endif
 build: ensure-submodule-initialized grpc/buf build-module
 	cd viam-cartographer && cmake -Bbuild -G Ninja ${EXTRA_CMAKE_FLAGS} && cmake --build build
 
+# Ideally build-asan would be added build-debug, but can't yet 
```

Comment:
[nit] would be added -> would be added to

Reply:
done